src/components/DeviceConfirm/index.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+
+import { rgba } from 'styles/helpers'
+
+import Box from 'components/base/Box'
+
+import IconCheck from 'icons/Check'
+import IconCross from 'icons/Cross'
+
+const pulseAnimation = p => keyframes`
+  0% {
+    box-shadow: 0 0 0 1px ${rgba(p.theme.colors.wallet, 0.4)};
+  }
+  70% {
+    box-shadow: 0 0 0 6px ${rgba(p.theme.colors.wallet, 0)};
+  }
+  100% {
+    box-shadow: 0 0 0 0 ${rgba(p.theme.colors.wallet, 0)};
+  }
+`
+
+const Wrapper = styled(Box).attrs({
+  color: p => (p.notValid ? 'alertRed' : 'wallet'),
+  relative: true,
+})`
+  padding-top: ${p => (p.notValid ? 0 : 30)}px;
+  transition: all ease-in-out 0.1s;
+`
+
+const WrapperIcon = styled(Box)`
+  color: ${p => (p.notValid ? p.theme.colors.alertRed : p.theme.colors.positiveGreen)};
+  position: absolute;
+  left: 193px;
+  bottom: 16px;
+
+  svg {
+    transition: all ease-in-out 0.1s;
+  }
+`
+
+const Check = ({ notValid }: { notValid: boolean }) => (
+  <WrapperIcon notValid={notValid}>
+    {notValid ? <IconCross size={11} /> : <IconCheck size={11} />}
+  </WrapperIcon>
+)
+
+const PushButton = styled(Box)`
+  background: linear-gradient(to bottom, #ffffff, ${p => p.theme.colors.wallet});
+  bottom: 48px;
+  height: 28px;
+  left: 205px;
+  position: absolute;
+  width: 1px;
+
+  &:before {
+    animation: ${p => pulseAnimation(p)} 1s linear infinite;
+    background-color: ${p => p.theme.colors.wallet};
+    border-radius: 50%;
+    bottom: 0;
+    box-sizing: border-box;
+    content: ' ';
+    display: block;
+    height: 9px;
+    left: 50%;
+    margin-bottom: -4px;
+    margin-left: -4px;
+    position: absolute;
+    width: 9px;
+    z-index: 1;
+  }
+`
+
+type Props = {
+  notValid: boolean,
+}
+
+const DeviceConfirm = (props: Props) => (
```

Comment:
minor: It can be good to make this a pure component OR to put the static `<svg>` element into a constant so React don't have to work a lot